Resources/Prototypes/_Mono/Shipyard/Expedition/picses.yml

```
@@ -0,0 +1,40 @@
+# Author Info
+# GitHub: starch70
+# Discord: _starch_
+
+# Shuttle Notes: its the goob nukeops shuttle exterior (docking ports are a bit different)
+#
+- type: vessel
+  id: Picses
+  parent: BaseVessel
+  name: TSK Picses
+  description: A compact but capable expedition ship, with durable hull plating too.
+  price: 85250 # plastitanium tax...
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)